frontend/public/components/limit-range.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+// eslint-disable-next-line no-unused-vars
+import {ColHead, DetailsPage, List, ListHeader, ListPage} from './factory';
+import {K8sResourceKindReference} from "../module/k8s";
+import {Cog, navFactory, Heading, ResourceCog, ResourceLink, ResourceSummary, Timestamp} from "./utils";
+import {registerTemplate} from "../yaml-templates";
+
+const { common } = Cog.factory;
+
+const menuActions = [
+  ...common,
+];
+
+registerTemplate('v1.LimitRange', `apiVersion: v1
+kind: LimitRange
+metadata:
+  name: mem-limit-range
+spec:
+  limits:
+  - default:
+      memory: 512Mi
+    defaultRequest:
+      memory: 256Mi
+    type: Container
+`);
+
+const LimitRangeReference: K8sResourceKindReference = 'LimitRange';
+
+const LimitRangeRow: React.SFC<LimitRangeRowProps> = ({obj}) =>
+  <div className="row co-resource-list__item">
+    <div className="col-xs-3">
+      <ResourceCog actions={menuActions} kind={LimitRangeReference} resource={obj} />
+      <ResourceLink kind={LimitRangeReference} name={obj.metadata.name} namespace={obj.metadata.namespace}
+                    title={obj.metadata.resourceType} />
+    </div>
+    <div className="col-xs-3">
+      <Timestamp timestamp={obj.metadata.creationTimestamp} />
+    </div>
+  </div>;
+
+const LimitRangeHeader = props =>
+  <ListHeader>
+    <ColHead {...props} className="col-xs-3" sortField="metadata.name">NAME</ColHead>
```

Comment:
Need a namespace column after name (so you know what namespace it's from when looking at limit ranges for all projects)

Reply:
@nicolethoen Ex: https://github.com/openshift/console/blob/master/frontend/public/components/routes.tsx#L149